### PR TITLE
[MAIL-95] mailer history icons

### PR DESCRIPTION
**Pull request by BBQlettuce**
update svgs for `IconMailerHistorySolid` and `IconMailerHistoryOutline`

### CODE REVIEW

File: src/ionicons/solidIcons/IconMailerHistorySolid.tsx [hunk 1]
```diff
@@ -4,12 +4,25 @@ import SvgIcon, { SvgIconProps } from '@material-ui/core/SvgIcon';
...
+          fillRule="evenodd"
+          clipRule="evenodd"
+          d="M17.6137 22.6702C20.3922 22.6702 22.6446 20.4178 22.6446 17.6393C22.6446 14.8608 20.3922 12.6084 17.6137 12.6084C14.8352 12.6084 12.5828 14.8608 12.5828 17.6393C12.5828 20.4178 14.8352 22.6702 17.6137 22.6702ZM18.0359 15.3114C18.0359 14.8972 17.7001 14.5614 17.2859 14.5614C16.8717 14.5614 16.5359 14.8972 16.5359 15.3114V17.8114C16.5359 18.2256 16.8717 18.5614 17.2859 18.5614H19.7859C20.2001 18.5614 20.5359 18.2256 20.5359 17.8114C20.5359 17.3972 20.2001 17.0614 19.7859 17.0614H18.0359V15.3114Z"
+        />
+      </g>
+      <defs>
+        <clipPath id="clip0">
+          <rect width="24" height="24" transform="translate(0 0.20459)" />
+        </clipPath>
+      </defs>
```
Comment (herrethan) [inline src/ionicons/solidIcons/IconMailerHistorySolid.tsx:25]: this `clipPath` is an odd inclusion. can you check if you are getting the latest one?
https://www.figma.com/file/yFuEsO8UMFArgoJHhtF6T0/The-Liminal-Project?node-id=800%3A10881

from what im seeing, its just two filled paths...

Comment (BBQlettuce) [reply src/ionicons/solidIcons/IconMailerHistorySolid.tsx:25]: looks like that link doesn't have the right icon... i pulled from https://www.figma.com/file/gL4ne4FElhPkB5wXFZIoLp/Material-Design-for-Reonomy?node-id=19660%3A37548

Comment (herrethan) [reply src/ionicons/solidIcons/IconMailerHistorySolid.tsx:25]: check on that file, i think yours is the older version. the liminal project as far as i understand it is the definitive.
this icon right? 
image

Comment (BBQlettuce) [reply src/ionicons/solidIcons/IconMailerHistorySolid.tsx:25]: @herrethan sorry bout the confusion. I confirmed with Cathy, the one on liminal isn't up to date, and the one we are trying to add is the 'mailbox' looking one on figma. should end up with this in the app:
Screen Shot 2020-12-16 at 1 48 26 PM

Comment (BBQlettuce) [reply src/ionicons/solidIcons/IconMailerHistorySolid.tsx:25]: to be clear, we are only using the "outlined" version (i.e. the other file that was changed in this PR). I went ahead and updated the solid version too but we could just leave this file alone if we wanted to. I'm not sure about the `clipPath` stuff either, just how it ended up when i "copied as SVG" from figma 🤷

Comment (herrethan) [reply src/ionicons/solidIcons/IconMailerHistorySolid.tsx:25]: the `clipPath` is the red flag that says to me this is the older file.
we should also rename these `IconMailboxOutline` `IconMailboxSolid` in that case.
"mailerHistory" is a different icon.

Comment (BBQlettuce) [reply src/ionicons/solidIcons/IconMailerHistorySolid.tsx:25]: ^ these are the mailer history icons (they have the clock). `IconMailbox_` exists already, looks like this but without the clock

Comment (BBQlettuce) [reply src/ionicons/solidIcons/IconMailerHistorySolid.tsx:25]: 🤔  interestingly, when i copy what looks like the same icon in liminal, it does not have the clip paths. i'll update this file

**Review by herrethan — APPROVED** [review]
there tis!

**Pull request merged**